packages/core/test/lib/envelope.test.ts

```diff
@@ -95,6 +95,13 @@ describe('createSpanEnvelope', () => {
     client = new TestClient(options);
     setCurrentClient(client);
     client.init();
+
+    // We want to avoid console errors in the tests
+    jest.spyOn(console, 'error').mockImplementation(() => {});
```

Comment:
Finally added this because it kept annoying me when running these tests locally to get this test log output all the time 😅

Reply:
Actually I'll move this out to https://github.com/getsentry/sentry-javascript/pull/14358